client/v3/naming/endpoints/endpoints_impl.go

```diff
@@ -0,0 +1,121 @@
+package endpoints
+
+// TODO: The API is not yet implemented.
```

Comment:
Can we move this as backlogged GitHub issue? Or put these into `internal` package? Once we have this public, it's hard to remove. Seems like this has not been implemented?

Reply:
Added the issue: 
  https://github.com/etcd-io/etcd/issues/12652
I assume this will get implemented as part of 3.5.0 (I will contribute if none else).

Reply:
@ptabor Thanks!